libs/parallelism/algorithms/include/hpx/parallel/container_algorithms/remove.hpp

```diff
@@ -464,7 +463,7 @@ namespace hpx {
         subrange_t<typename hpx::traits::range_iterator<Rng>::type>>::type
     remove_if(ExPolicy&& policy, Rng&& rng, Pred&& pred, Proj&& proj = Proj());
 
-}    // namespace hpx
+}}    // namespace hpx::ranges
```

Comment:
Could we change the use of `tag_invoke` below to `tag_invoke_fallback`, please? This could however be done in a separate PR (please turn this comment into a separate ticket if needed).

Reply:
@hkaiser I am confused. Which part are you referring to exactly? I do not use `tag_invoke` in this PR at all.